ush/forecast_postdet.sh

```diff
@@ -850,10 +857,12 @@ MOM6_nml() {
 
 MOM6_out() {
   echo "SUB ${FUNCNAME[0]}: Copying output data for MOM6"
+  mkdir -p "${COM_CONF}"
 
   # Copy MOM_input from DATA to COM_OCEAN_INPUT after the forecast is run (and successfull)
   if [[ ! -d ${COM_OCEAN_INPUT} ]]; then mkdir -p "${COM_OCEAN_INPUT}"; fi
-  ${NCP} "${DATA}/INPUT/MOM_input" "${COM_OCEAN_INPUT}/"
+  ${NCP} "${DATA}/INPUT/MOM_input" "${COM_OCEAN_INPUT}/" # TODO: Are both needed?
```

Comment:
```suggestion
```

Reply:
Done.

Reply:
What do you mean by both in the TODO statement?

Reply:
Oh now I get it.  I don't think COM_OCEAN_INPUT is needed now that it's in COM_CONF